### DISCUSSION TITLE
Qual a melhor forma de entregar um site para um cliente?

### DISCUSSION THREAD

**Discussion by thiamsantos** [frontendbr/forum#269] [Perguntas e Respostas]
Qual a forma de entregar um site para um cliente totalmente leigo possa editar o conteúdo(texto). Sem eu precisar ter que mudar muito meu workflow, trabalhar com algo próximo com uma template engine.

Quais seriam as opções?

**Comment by alanhr** [comment]
você pode usar algum CMS  tipo:
[Grav](https://getgrav.org/), [October](https://octobercms.com/) ou Wordpress

**Comment by kazzkiq** [comment]
Recentemente eu estava procurando um editor de texto para meu projeto e acabei achando uma biblioteca open-source muito legal que te permite editar o layout da sua página pelo próprio site real!

[![](https://camo.githubusercontent.com/163aed149d2547b433b9ee964b5d914a5d030823/687474703a2f2f676574636f6e74656e74746f6f6c732e636f6d2f696d616765732f6769746875622d73706c6173682e706e67)](http://getcontenttools.com/demo)

Você clica no ícone vermelho no canto esquerdo da tela e consegue ir editando os textos do seu layout e já vendo como ficaria em tempo real. Apesar de não ter testado na prática, achei o plugin muito interessante e definitivamente usarei ele para freelas futuros. Espero que seja útil no seu caso. :)

Demo: http://getcontenttools.com/demo
Repositório: https://github.com/GetmeUK/ContentTools
